LICENSE.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015, Jesse Berger (user alias: valadian)
```

Comment:
I've seen other open source projects revert the work of particular authors who refused to accept a license change, with other members rewriting it in different ways.

I'd say it's dubious legally when its a core author; easier to reach out to them and get their consent (the author of CivModCore is still around, I do believe members of the admin team have direct contact).

Otherwise yes, we'd have to completely rewrite it, preferably from a very different architectural standpoint so any claim of direct inspiration was clearly void.